### ISSUE TITLE
Ability to specify image icon

### ISSUE THREAD

**Issue by martinbean** [dylang/grunt-notify#93]
It would be great if there was an option to specify the image icon displayed in notifications instead of the default Grunt logo. I did look in the documentation for this but couldn’t see it. Apologies if I’ve missed it and that this is already possible.

**Comment by martinbean** [comment]
Bump.

**Comment by honsa** [comment]
A pull request is there, waiting to approve. 

It looks like there is no interest to invest some time in this nice little grunt plugin.

https://github.com/dylang/grunt-notify/pull/96

**Comment by martinbean** [comment]
@honsa Yeah, it’s a shame. I’m guessing this project is dead—the last commit to `master` was nearly three years ago now, despite having open issues and pull requests.

@dylang Are you still maintaining this plugin?
